docs/src/API Reference/ext/index.md

```diff
@@ -5,6 +5,9 @@ These files contain useful features that help you develop a bot
 - [Debug Extension](debug_ext)
     - An extension preloaded with a load of debugging utilities to help you find and fix bugs
 
+- [Jurigged](jurigged)
```

Comment:
This name means nothing to people who don't already know the library.  I'd rather call it "Live Editing" or "Hot Reloading" or something similar.

Reply:
I think "Live Patching" is a better name